src/gen_dragonfruit/gd_dev_adafruit_MPL115A2_press.cpp

```diff
@@ -7,10 +7,10 @@ void gd_dev_adafruit_MPL115A2_press_open(void){
     mpl115a2t1_press.begin();
 }
 
-uint16_t gd_dev_adafruit_MPL115A2_press_read(void){
-  uint16_t value = 0;
+uint32_t gd_dev_adafruit_MPL115A2_press_read(void){
+  float value = 88;
   #ifndef SEN_STUB
-  value = mpl115a2t1_press.getPressure()*1000;
+  value = mpl115a2t1_press.getPressure()*1000; //Function gives kPa, x1000 for Pa
```

Comment:
Two things:

* There should be a space after the `//`
* The comment description could be a little better. What gives kPa? And why are you converting to Pa?